extensions/azure/data-plane-azure-storage/src/test/java/org/eclipse/dataspaceconnector/azure/dataplane/azurestorage/adapter/BlobAdapterFactoryTest.java

```diff
@@ -0,0 +1,21 @@
+package org.eclipse.dataspaceconnector.azure.dataplane.azurestorage.adapter;
+
+import org.junit.jupiter.api.Test;
+
+import static org.eclipse.dataspaceconnector.azure.dataplane.azurestorage.pipeline.AzureStorageTestFixtures.createAccountName;
+import static org.eclipse.dataspaceconnector.azure.dataplane.azurestorage.pipeline.AzureStorageTestFixtures.createBlobName;
+import static org.eclipse.dataspaceconnector.azure.dataplane.azurestorage.pipeline.AzureStorageTestFixtures.createContainerName;
+import static org.eclipse.dataspaceconnector.azure.dataplane.azurestorage.pipeline.AzureStorageTestFixtures.createSharedKey;
+
+class BlobAdapterFactoryTest {
+
+    @Test
+    void getBlobAdapter_succeeds() {
+        BlobAdapterFactory f = new BlobAdapterFactory(null);
+        f.getBlobAdapter(
+                createAccountName(),
+                createContainerName(),
+                createBlobName(),
+                createSharedKey());
+    }
```

Comment:
it doesn't assert anything. What about using:

```
assertThatNoException()
                .isThrownBy(() -> f.getBlobAdapter(
                    createAccountName(),
                    createContainerName(),
                    createBlobName(),
                    createSharedKey()));
```
or checking the type of returned blob?

```
assertThat(f.getBlobAdapter(
                createAccountName(),
                createContainerName(),
                createBlobName(),
                createSharedKey())).isInstanceOf(DefaultBlobAdapter.class);
```

Reply:
Well it does verify that no exceptions are thrown. But the syntax you propose makes it clearer - adapted